docs/connectors/twitch.md

```diff
@@ -0,0 +1,78 @@
+# Twitch
+
+A connector for [Twitch](https://twitch.tv/).
+
+## Requirements
+
+- A Twitch Account
+- A Twitch App obtained from [Twitch developers page](https://dev.twitch.tv/console/apps)
+- The `code` obtained from the first step of OAuth
+
+## Configuration
+
+```yaml
+connectors:
+  twitch:
+    # required
+    code: "hfu923hfks02nd2821jfislf"  # Code obtained from the first oauth step
+    client-id: "e0asdj48jfkspod0284"
+    client-secret: "kdksd0458j93847j"
+    channel: theflyingdev # Broadcaster channel
+    redirect: http://localhost # Url to be passed to get oath token - defaults to localhost
+    foward-url: 'http://94jfsd9ff.ngrok.io' # Either an URL provided by a forwarding service or an exposed ip address
```

Comment:
In the GitHub connector rewrite I've added a `base-url` configuration option to the `web` config, so I would quite like to use that consistently between connectors. More and more connectors are moving to this model so it would be good to configure it in one place.

```yaml
web:
  base-url: https://url-to-opsdroid.com
```

Reply:
To be clear this isn't done yet.